charts/newrelic-infrastructure-beta/README.md

```diff
@@ -0,0 +1,208 @@
+# newrelic-infrastructure
```

Comment:
For the new values, we could consider using this https://github.com/norwoodj/helm-docs